proto/interchain_security/ccv/provider/v1/query.proto

```diff
@@ -73,6 +73,15 @@ service Query {
     option (google.api.http).get =
         "/interchain_security/ccv/provider/registered_consumer_reward_denoms";
   }
+
+  // QueryProposedConsumerChainIDs returns the chain IDs of the proposed consumer chain addition proposals
```

Comment:
IDNK why git's seeing this change. The protos aren't changed from the feature branch.
https://github.com/cosmos/interchain-security/blob/feat/refactor-key-assignment/proto/interchain_security/ccv/provider/v1/query.proto#L79-L86